lib/service/Service.ts

```diff
@@ -40,11 +40,75 @@ class Service {
   }
 
   /**
-   * Placeholder for a method to return general information about an Exchange Union node.
+   * Get general information about this Exchange Union node.
    */
   public getInfo = async () => {
-    const lnd = await this.lndClient.getInfo();
-    return { lnd };
+    const info: any = {};
+
+    info.version = Xud.getVersion();
+
+    const peers = this.pool.peers;
+    const pairs = await this.orderBook.getPairs();
+    info.peers = peers.length();
+    info.pairs = pairs.length;
+
+    let peerOrdersCount: number = 0;
+    let ownOrdersCount: number = 0;
+    for (const key in pairs) {
+      const pair = pairs[key];
+
+      const orders = await this.orderBook.getOrders(pair.id, 0);
+      const ownOrders = await this.orderBook.getOwnOrders(pair.id, 0);
```

Comment:
This seems like a computationally expensive loop. Maybe not that big of a deal once we have proper caching and performance optimizations - but I'm wondering if it would be better to explicitly track the number of all orders for informational purposes like this. Then we don't need any loop or queries here. Thoughts? @moshababo